src/sentry/eventstore/processing/base.py

```diff
@@ -22,9 +24,9 @@ class BaseEventProcessingStore:
     implementations.
     """
 
-    def __init__(self, inner, timeout: int = DEFAULT_TIMEOUT):
-        self.inner = inner
-        self.timeout = timeout
+    def __init__(self, store: KVStorage[str, Event], timeout: int = DEFAULT_TIMEOUT):
+        self.store = store
```

Comment:
`mypy` would have told me about this, had I spent the time to get it running correctly in my sentry workspace in VSCode:

```
src/sentry/eventstore/processing/base.py:28: error: Cannot assign to a method
src/sentry/eventstore/processing/base.py:28: error: Incompatible types in assignment (expression has type "KVStorage[str, Any]", variable has type "Callable[[Any, bool], str]")
src/sentry/eventstore/processing/base.py:38: error: "Callable[[Any, bool], str]" has no attribute "set"
src/sentry/eventstore/processing/base.py:44: error: "Callable[[Any, bool], str]" has no attribute "get"
src/sentry/eventstore/processing/base.py:47: error: "Callable[[Any, bool], str]" has no attribute "delete"
src/sentry/eventstore/processing/base.py:48: error: "Callable[[Any, bool], str]" has no attribute "delete"
```

🤦